src/lib/swapper/swappers/ThorchainSwapper/getThorTradeQuote/getTradeQuote.ts

```diff
@@ -112,8 +121,12 @@ export const getThorTradeQuote = async (
       return getL1quote(input, streamingInterval, tradeType)
     case TradeType.LongTailToL1:
       return getLongtailToL1Quote(input, streamingInterval, assetsById)
-    case TradeType.LongTailToLongTail:
```

Comment:
Is this still valid and something we want to keep?

Reply:
No sure to understand? I did keep it returning an error monad right after on line 127, we keep it as it can be a valid feature we wan't in the future, Longtail to longtail can be easily done by wiring things up even if it's super best effort

Reply:
It is still valid @gomesalexandre, we'll want to eventually handle full long-tail to long-tail trades 🤝